cloud/ec2_fleet_test.go

```diff
@@ -55,6 +57,17 @@ func TestFleet(t *testing.T) {
 			assert.NoError(t, err)
 			assert.Equal(t, "us-east-1a", hDb.Zone)
 		},
+		"GetInstanceStatusNonExistent": func(*testing.T) {
+			aws_error := awserr.New(EC2ErrorNotFound, "The instance ID 'test-id' does not exist", nil)
+			wrapped_aws_error := errors.Wrap(
```

Comment:
It shouldn't be necessary to wrap this twice since we assume that `errors.Cause` behaves as expected with `errors.Wrap`. Just one `errors.Wrap` should be sufficient.